scripts/merge-and-build.sh

```diff
@@ -114,6 +139,8 @@ else
   fi
 fi # End check if we are master
 
+# TODO: Origin already vendors the web console. Should we stop updating it separately for OSE?
```

Comment:
I just realized that your `Web client branding` commit is different from the one I referred to in my initial comment. So we can end up with something like:
```
# x--x--x--m--o--w--s--b--t
```
where `b` is a bump commit for the web console (additional diff between console HEAD and the vendored console in Origin).